docs/_posts/2017-06-14-jekyll-3-5-0-released.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Jekyll turns 3.5, oh my!'
+date: 2017-06-15 17:32:32 -0400
+author: parkr
+version: 3.5.0
+categories: [release]
+---
+
+Good news! Nearly 400 commits later, Jekyll 3.5.0 has been released into
+the wild. Some new shiny things you might want to test out:
+
+- Jekyll now uses Liquid 4, the latest!
```

Comment:
@adamvoss A commit regarding [`whitespace control`](https://github.com/Shopify/liquid/commit/e1d40c7d89accd91f0d3f9bba0637ee33f708646).
The meta info above the author confirms its existence in `4.0-stable`